marketplace/deployer_util/wait_for_ready.py

```diff
@@ -200,6 +205,10 @@ def labels(resource):
   return resource['metadata']['labels']
 
 
+def annotations(resource):
+  return resource['metadata']['annotations']
```

Comment:
use `get('metadata', {}).get('annotations', {})` to be safe? I know we should do that everywhere too, but at least this is an improvement. We can eventually change the other places as well.